tests/test_pidfile.py

```diff
@@ -15,7 +15,7 @@
 
 
 def builtin(name):
-    if sys.version_info >= (3, 0):
+    if sys.version_info[0] >= 3:
```

Comment:
Since we are going to drop the if statement soon (when we drop the Python 2 support), I'd prefer leaving this as-is.

Reply:
✅